unet/utils.py

```diff
@@ -0,0 +1,95 @@
+from tensorflow import keras
+import numpy as np
+
+# from keras.preprocessing.image import array_to_img
+from math import floor
+from PIL import Image, ImageOps
+from skimage.transform import resize
+from typing import Optional, Tuple
+
+
+def infer(model, img : Image) -> np.ndarray:
```

Comment:
thoughts on my opening a PR to extend this where we create a class that initializes with a specific model, then we can call a class function to process an image?  should let us process multiple images more efficiently using the same model.  Do we think that would be helpful for us?

Reply:
Sure, you can do that, but I don't think we have time to do another model at the moment.